code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -171,3 +171,54 @@
 	. = ..()
 	new /obj/item/storage/surgical_tray(src)
 	new /obj/item/roller/surgical(src)
+
+/obj/structure/closet/secure_closet/professor_dummy
+	name = "professor dummy cabinet"
+	desc = "An ultrasafe cabinet containing Professor DUMMY and its tablet. Only accessible by Chief Medical Officers and Senior Listed Advisors."
+	icon_state = "surgical_wall_locked"
+	icon_closed = "surgical_wall_unlocked"
+	icon_locked = "surgical_wall_locked"
+	icon_opened = "surgical_wall_open"
+	icon_broken = "surgical_wall_spark"
+	health = null	// Unbreakable
+	unacidable = TRUE
+	unslashable = TRUE
+	store_mobs = TRUE
+	wall_mounted = TRUE
+
+/obj/structure/closet/secure_closet/professor_dummy/Initialize()
+	. = ..()
+	new /mob/living/carbon/human/dummy/professor_dummy(src)
+
+/obj/structure/closet/secure_closet/professor_dummy/togglelock(mob/living/user)
+	if(user.job == JOB_CMO || user.job == JOB_SEA)
+		..()
+		return
```

Comment:
```suggestion
		return ..()
```
Although nothing normally is returned, since you're returning anyways you may as well.